plugins/sitoa/common/Tools.h

```diff
@@ -89,16 +89,17 @@ namespace ATSTRING
    const AtString softmax("softmax");
    const AtString linkable("linkable");
    const AtString deprecated("deprecated");
-   // sitos-specific metadata
+   // sitoa-specific metadata
    const AtString soft_category("soft.category");
    const AtString soft_order("soft.order");
    const AtString soft_label("soft.label");
    const AtString soft_skip("soft.skip");
    const AtString soft_inspectable("soft.inspectable");
    const AtString soft_viewport_guid("soft.viewport_guid");
    const AtString soft_node_type("soft.node_type");
-   //
+   // sitoa custom ports/types
    const AtString closure("closure");
+   const AtString operator_node("operator");
```

Comment:
Is this used now or could it be removed?
```suggestion
```

Reply:
Ya, this should be removed, I was having trouble getting a scripted shader plugin working and thought it needed to have a c++ counter part. This is left over from that experiment...

Reply:
Actually, this is a remnant from when we started implementing shader def support for operators. Still, I did a quick find in files for the entire code base and didn't see anywhere it was used.